packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -260,29 +260,65 @@ function filterSettingsToParams(filterSettings: FilterSettings): any {
   for (let tag of tagsExcluded) {
     tagsFilter[`tagRelevance.${tag.tagId}`] = {$not: {$gte: 1}};
   }
+
+  /**
+   * For example:
+   * {
+   *   tagId: <Community tag ID>,
+   *   scoreMultiplier: 0.1
+   * }
+   * Will do:
+   * All posts with the Community tag will get their score multiplied by 0.1 for purposes of sorting front-page posts
+   * 
+   * TODO: Move to DB (or to environment variable?) (or to another file?)
+   */
+  const tagScoreMultipliers = [
+    {
+      tagId: "ZCihBFp5P64JCvQY6", // Community
+      scoreMultiplier: 0.5
+    },
+  ]
   
   const tagsSoftFiltered = _.filter(filterSettings.tags, t => (t.filterMode!=="Hidden" && t.filterMode!=="Required" && t.filterMode!=="Default" && t.filterMode!==0));
   let scoreExpr: any = null;
   if (tagsSoftFiltered.length > 0 || frontpageSoftFilter.length > 0) {
     scoreExpr = {
       syntheticFields: {
-        score: {$divide:[
-          {$add:[
-            "$baseScore",
-            ...tagsSoftFiltered.map(t => ({
+        score: {
+          $divide: [
+            {
               $multiply: [
```

Comment:
"multiply" was added to multiply the scores.
Also possible: to use the "$divide" that already exists, but it might be a bit less readable.
[no idea if adding an extra function here causes performance problems in mongo, or is bad practice for some other reason]